assets/html/index.html

```diff
@@ -6,10 +6,10 @@
     <!-- bootstrap -->
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
     <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
-    <link href="assets/fonts/font-awesome/css/font-awesome.min.css" rel="stylesheet">
-    
-    <link rel="stylesheet" type="text/css" href="styles.css">
-   
+
+    <link rel="stylesheet" href="../../bower_components/font-awesome/css/font-awesome.min.css">    
```

Comment:
kshtu nuk guxon me ju referu css-it, edhe per qka je tu e ba minify nese i referohesh qeshtu?